portal/server/controllers/utilisation-report-service/previous-reports/index.js

```diff
@@ -0,0 +1,49 @@
+const utilisationData = require('./data.json');
+
+const getMonthName = (monthNumber) => {
+  const date = new Date();
+  // offset by 1 as January = 1 in Database
+  date.setMonth(monthNumber - 1);
+  return date.toLocaleString('default', { month: 'long' });
+};
+
+const getUtilisationReportDownload = async (req, res) => {
+  // get data from the DB (currently using json) - will need to pass in bank id from user
+  try {
+    const { targetYear } = req.query;
+    const navItems = utilisationData.map((utilisation) => ({
+      text: utilisation.year,
+      href: `?targetYear=${utilisation.year}`,
+      attributes: { 'data-cy': `side-navigation-${utilisation.year}` },
+    }));
+
+    const utilisation = targetYear ? utilisationData.find((x) => x.year.toString() === targetYear) : utilisationData[0];
```

Comment:
Leaving a note that this explicitly needs backend to order the data in descending year, nothing wrong with that just for if I look later!

Reply:
Ah yes, I've made a note that the backend should be ordered by descending year and ascending month.
Additionally the backend will return an empty array for years where no reports were submitted but were submitted for years before AND after this year